files/default/kagent_utils/tests/test_kagent_config.py

```diff
@@ -68,7 +68,6 @@ def test_parse_full_config(self):
         self.assertEqual(self._toUrl(self.path_ca_host), config.ca_host_url)
         self.assertEqual(self._toUrl(self.path_heartbeat),
                          config.heartbeat_url)
-        self.assertEqual(self._toUrl(self.path_alert), config.alert_url)
```

Comment:
You should also remove other occurrences `path_alert`

Reply:
done